snuba/clickhouse/http.py

```diff
@@ -92,7 +80,7 @@ def write(self, rows: Iterable[WriterTableRow]):
             details = CLICKHOUSE_ERROR_RE.match(content)
             if details is not None:
                 code, type, message = details.groups()
-                raise ClickHouseError(int(code), type, message)
+                raise ClickhouseError(int(code), message)
```

Comment:
What did type provide ?

Reply:
I only ever saw it as `DB::Exception` which didn't seem to provide much value.